api/src/main/java/org/openmrs/module/openconceptlab/importer/Saver.java

```diff
@@ -603,13 +608,16 @@ private Item updateOrAddAnswersFromOcl(Import update, OclMapping oclMapping, Con
 			}
 		}
 
-		if (!found) {
-			ConceptAnswer conceptAnswer = new ConceptAnswer(answer);
-			conceptAnswer.setUuid(oclMapping.getExternalId());
-			conceptAnswer.setSortWeight(getSortWeightForMapping(oclMapping, null));
-			question.addAnswer(conceptAnswer);
-			item = new Item(update, oclMapping, ItemState.ADDED);
-		}
+		if (!found)
+			if(BooleanUtils.isNotTrue(oclMapping.getRetired())) {
+				ConceptAnswer conceptAnswer = new ConceptAnswer(answer);
+				conceptAnswer.setUuid(oclMapping.getExternalId());
+				conceptAnswer.setSortWeight(getSortWeightForMapping(oclMapping, null));
+				question.addAnswer(conceptAnswer);
+				item = new Item(update, oclMapping, ItemState.ADDED);
+			} else {
+				item = new Item(update, oclMapping, ItemState.UP_TO_DATE);
```

Comment:
This seems like the correct state to report here, but interested in others thoughts.

Reply:
Seems right. `UP_TO_DATE` is intended to mark exactly this, I think.